client/apps/settings/stylesheets/promoted_admin.styl

```diff
@@ -0,0 +1,18 @@
+.promoted-admin
+  &__section-header
+    display flex
+    align-items center
+    border-bottom 1px solid gray-color
+    margin-top 30px
+    padding-bottom 10px
+    h1
+      Garamond()
+      font-size 30px
+      padding-right 20px
+    .dropdown-header__caret
+      display flex
+  &__section_title
+    Garamond()
```

Comment:
minor -- we usually do the classname with hyphens rather then underscore, as `&__section-title`